dc-cudami-admin/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/IdentifierTypeController.java

```diff
@@ -48,12 +37,6 @@ public String create() {
     return "identifiertypes/create";
   }
 
-  @GetMapping("/api/identifiertypes/new")
-  @ResponseBody
-  public IdentifierType createModel() throws TechnicalException {
-    return service.create();
-  }
-
   @GetMapping("/identifiertypes/{uuid}/edit")
   public String edit(@PathVariable UUID uuid, Model model) throws TechnicalException {
     IdentifierType identifierType = service.getByUuid(uuid);
```

Comment:
Müßte hier nicht, analog zur unten stehenden `view`-Methode, eine ResourceNotFoundException geworfen werden, wenn getByUuid() null zurückliefert? Andernfalls würde es spätestens in der folgenden Zeile zu einer NullpointerException kommen.

Reply:
ja, hab ich ergänzt